### PR TITLE
Updated secret scanning

### PR DESCRIPTION
**Pull request by gmsdelmundo**

### CODE REVIEW

File: .github/workflows/shared-java-publish-versioned-package.yaml [hunk 1]
```diff
@@ -30,38 +33,39 @@ jobs:
...
           scan-type: 'fs'
-          format: 'sarif'
+          format: 'table'
           exit-code: '1'
           ignore-unfixed: true
           severity: 'CRITICAL'
-          output: 'trivy-results.sarif'
           hide-progress: true
 
-      - name: Upload Trivy scan results to GitHub Security tab
```
Comment (thomasm-ttd) [inline .github/workflows/shared-java-publish-versioned-package.yaml:59]: Didn't mean to remove this completely, just remove the if: always() line

File: .github/workflows/shared-publish-docker-versioned.yaml [hunk 2]
```diff
@@ -105,34 +105,34 @@ jobs:
...
-          format: 'sarif'
+          scan-type: 'fs'
+          format: 'table'
           exit-code: '1'
           ignore-unfixed: true
           severity: 'CRITICAL'
-          output: 'trivy-results.sarif'
           hide-progress: true
 
-      - name: Upload Trivy scan results to GitHub Security tab
```
Comment (thomasm-ttd) [inline .github/workflows/shared-publish-docker-versioned.yaml:130]: We still need this don't we?

File: .github/workflows/shared-java-publish-versioned-package.yaml (hunk 1 above)
Comment (gmsdelmundo) [reply .github/workflows/shared-java-publish-versioned-package.yaml:59]: Oh it just got moved around. The `git diff` output may be a bit confusing, but here's the new code snippet:

```
      - name: Generate Trivy vulnerability scan report
        uses: aquasecurity/trivy-action@master
        if: inputs.publish_vulnerabilities
        with:
          scan-type: 'fs'
          format: 'sarif'
          exit-code: '0'
          ignore-unfixed: true
          severity: 'CRITICAL'
          output: 'trivy-results.sarif'
          hide-progress: true

      - name: Upload Trivy scan report to GitHub Security tab
        uses: github/codeql-action/upload-sarif@v2
        if: inputs.publish_vulnerabilities
        with:
          sarif_file: 'trivy-results.sarif'

      - name: Test with Trivy vulnerability scanner
        uses: aquasecurity/trivy-action@master
        with:
          scan-type: 'fs'
          format: 'table'
          exit-code: '1'
          ignore-unfixed: true
          severity: 'CRITICAL'
          hide-progress: true
```

File: .github/workflows/shared-release-major-minor-patch.yaml [hunk 3]
```diff
@@ -37,43 +40,44 @@ jobs:
...
         uses: aquasecurity/trivy-action@master
+        if: inputs.publish_vulnerabilities == 'true'
         with:
           scan-type: 'fs'
-          format: 'table'
+          format: 'sarif'
           exit-code: '0'
           ignore-unfixed: true
-          severity: 'CRITICAL,HIGH'
+          severity: 'CRITICAL'
```
Comment (thomasm-ttd) [inline .github/workflows/shared-release-major-minor-patch.yaml:51]: This should still be scanning at HIGH as well

File: .github/workflows/shared-release-major-minor-patch.yaml [hunk 4]
```diff
@@ -37,43 +40,44 @@ jobs:
...
           scan-type: 'fs'
-          format: 'sarif'
+          format: 'table'
           exit-code: '1'
           ignore-unfixed: true
           severity: 'CRITICAL'
-          output: 'trivy-results.sarif'
           hide-progress: true
 
-      - name: Upload Trivy scan results to GitHub Security tab
```
Comment (thomasm-ttd) [inline .github/workflows/shared-release-major-minor-patch.yaml:61]: Still need this

**Review by thomasm-ttd — APPROVED** [review]

**Pull request merged**